### PR TITLE
tinysearch 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/tinysearch.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "bin")
+    pkgshare.install "fixtures"
+  end
+
+  test do
+    output = shell_output("tinysearch #{pkgshare}/fixtures/index.json 2>&1", 1)
+    assert_match "Compiling WASM module using wasm-pack", output
+    assert_match "failed to execute \"wasm-pack\" \"build\"", output
```
Comment (carlocab) [inline Formula/tinysearch.rb:20]: Is this really expected? It doesn't seem like a good sign.

Comment (chenrui333) [reply Formula/tinysearch.rb:20]: I was thinking to add `wasm-pack` as test dependency, but it did not quite help with compilation, but I can use the tinysearch in my local just fine though.

Comment (carlocab) [reply Formula/tinysearch.rb:20]: You already have a `wasm-pack` dependency though. Is this really what you get from running the test case outside of `brew`?

Comment (cho-m) [reply Formula/tinysearch.rb:20]: Can you try setting up `rustup-init` like in `wasm-pack` formula?
```
    system "#{Formula["rustup-init"].bin}/rustup-init", "-y", "--no-modify-path"
    ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
```

Issue is that Rust/cargo isn't set up by the time testcase is running.

Comment (chenrui333) [reply Formula/tinysearch.rb:20]: I will give another spin today.

**Comment by chenrui333** [comment]
Should be in the better shape now.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**